packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -412,52 +474,6 @@ class PrebuiltIOSApp extends IOSApp {
   String get _bundlePath => bundleDir.path;
 }
 
-class ApplicationPackageStore {
```

Comment:
This does _almost_ the same thing as application package factory, it is just older and used for drive. It does not seem good to have two _almost_ the same classes